Core/include/Acts/Surfaces/RadialBounds.hpp

```diff
@@ -43,8 +44,8 @@ class RadialBounds : public DiscBounds {
   /// @param halfPhi The half opening angle (Pi for full angular coverage)
   /// @param avgPhi The average phi for the disc/ring sector
   RadialBounds(double minR, double maxR, double halfPhi = M_PI,
-               double avgPhi = 0.) noexcept(false)
-      : m_values({minR, maxR, halfPhi, avgPhi}) {
+               double avgPhi = 0., double angle = 0.) noexcept(false)
+      : m_values({minR, maxR, halfPhi, avgPhi, angle}) {
```

Comment:
I'm not sure I understand what the meaning of `angle` for `RadialBounds` is?

Reply:
Same answer as to Andi: This was a mistake, left over by a trial implementation of bevelling of the "side"-disc but in the end I didn't need it but I forgot to remove it